### ISSUE TITLE
IFFF support and TransferGroup

### ISSUE THREAD

**Issue by mhier** [ChimeraTK/DeviceAccess-DoocsBackend#8]
This ticket depends on #7. If the four (or some of the four) accessors of the same IFFF property are added to a TransferGroup, the transfer shall be merged into a single transfer. Data consistency can be ensured like this. Reading and writing shall be supported.

For information only: At first this will only work for poll-type transfers (i.e. no ZeroMQ). Later this can be extended for ZeroMQ transfers as well, but this is conceptionally not yet foreseen in DeviceAccess (see also #3). This shall be done in a separate ticket.
